woodwork/column_accessor.py

```diff
@@ -388,20 +389,69 @@ def set_semantic_tags(self, semantic_tags):
         """
         self._schema._set_semantic_tags(semantic_tags)
 
+    @_check_column_schema
+    def get_outliers(
```

Comment:
Now that we have two ways for calculating outliers, and under the possibility of adding more ways in the future, I wanted to create a general function that could retrieve the outliers information based on the method requested.

This would ideally act as the primary means by which to access outlier information.